be/CMakeLists.txt

```diff
@@ -614,6 +614,7 @@ set(STARROCKS_LINK_LIBS
     Types
     Udf
     Util
+    Script
```

Comment:
Could we use SQL instead of the script? I think the script it's not easy to use.

Reply:
It's hard to design & add a new set of imperative programming language syntax to SQL, like list/map/object, condition, for loop, etc, those would be a significant effort, so embedding a third-party scripting engine is more practical.